package.json

```diff
@@ -34,7 +34,7 @@
         "@openpgp/tweetnacl": "^1.0.3",
         "@openpgp/web-stream-tools": "^0.0.13",
         "jsmimeparser": "npm:@protontech/jsmimeparser@^3.0.1",
-        "openpgp": "npm:@protontech/openpgp@~5.9.1-0"
+        "openpgp": "larabr/openpgpjs#forwarding-generate-key-0x40"
```

Comment:
(Once https://github.com/ProtonMail/openpgpjs/pull/8 is merged, could you publish it and keep pointing to npm?)